tests/test_pipeline.py

```diff
@@ -337,187 +616,83 @@ def test_diad_testing_pipeline_output(
+    assert len(h5_files) == 1
 
-    serial_log_files = list(filter(lambda x: ".log" in x, serial_files))
-    parallel_log_files = list(filter(lambda x: ".log" in x, parallel_files))
-    assert len(serial_log_files) == 1
-    assert len(parallel_log_files) == 1
+    log_files = list(filter(lambda x: ".log" in x, files))
+    assert len(log_files) == 1
 
-    log_contents = _get_log_contents(serial_log_files[0])
-    mpi_log_contents = _get_log_contents(parallel_log_files[0])
+    log_contents = _get_log_contents(log_files[0])
 
-    assert "DEBUG | The full dataset shape is (220, 128, 160)" in log_contents
+    assert "The full dataset shape is (3201, 22, 26)" in log_contents
     assert (
-        "DEBUG | RANK: [0], Data shape is (180, 128, 160) of type uint16"
-        in log_contents
+        "Loading data: tests/test_data/k11_diad/k11-18014.nxs" in log_contents
+    )
+    assert "Path to data: /entry/imaging/data" in log_contents
+    assert "Preview: (100:3101, 8:15:, :)" in log_contents
+    assert (
+        "Data shape is (3001, 7, 26) of type uint16" in log_contents
     )
-    assert "DEBUG | Preview: (0:180, :, :)" in log_contents
-    assert "INFO | ~~~ Pipeline finished ~~~" in log_contents
-
-    #: user log and mpi log would differ in the data shapes
     assert (
-        "DEBUG | RANK: [0], Data shape is (45, 128, 160) of type uint16"
-        in mpi_log_contents
+        "Saving intermediate file: 5-httomolibgpu-FBP-tomo.h5" in log_contents
     )
-"""
+    assert (
+            "Global min -0.00561" in log_contents
+        )
+    assert (
+            "Global max 0.006643" in log_contents
+        )
 
-@pytest.mark.preview
-def test_sweep_range_pipeline_with_step_absent(
-    cmd, standard_data, sample_pipelines, output_folder
-):
-    cmd.insert(7, standard_data)
-    cmd.insert(8, sample_pipelines + "testing/step_absent.yml")
-    subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
+# @pytest.mark.preview
+# def test_sweep_range_pipeline_with_step_absent(
+#     cmd, standard_data, sample_pipelines, output_folder
+# ):
+#     cmd.insert(7, standard_data)
+#     cmd.insert(8, sample_pipelines + "testing/step_absent.yml")
+#     subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
 
-    log_files = list(filter(lambda x: ".log" in x, read_folder("output_dir/")))
-    assert len(log_files) == 1
+#     log_files = list(filter(lambda x: ".log" in x, read_folder("output_dir/")))
+#     assert len(log_files) == 1
 
-    log_contents = _get_log_contents(log_files[0])
+#     log_contents = _get_log_contents(log_files[0])
 
-    assert (
-        "ERROR | Please provide `start`, `stop`, `step` values"
-        " when specifying a range to peform a parameter sweep over."
-    ) in log_contents
+#     assert (
+#         "ERROR | Please provide `start`, `stop`, `step` values"
+#         " when specifying a range to peform a parameter sweep over."
+#     ) in log_contents
```

Comment:
If we're not interested in thinking about sweep functionality right now, it may be better to just remove this entirely rather than commenting out (other commented-out sweep tests like `test_sweep_pipeline_with_save_all_using_mpi` have been removed in this PR anyway)